transportable-udfs-presto/src/main/java/com/linkedin/transport/presto/PrestoWrapper.java

```diff
@@ -54,10 +63,16 @@ public static StdData createStdData(Object prestoData, Type prestoType, StdFacto
       return new PrestoInteger(((Long) prestoData).intValue());
     } else if (prestoType instanceof BigintType) {
       return new PrestoLong((long) prestoData);
-    } else if (prestoType.getJavaType() == boolean.class) {
```

Comment:
If I remember correctly, this used to handle Unknown/Null type. Could you please make sure that is still handled as expected?

Reply:
Can you elaborate? Do you mean Unknown/Null type has javaType of boolean.class?

In the method `createStdType()`, we also did `instance of ` check. So my change is just to make both methods consistent.

Also how can I make sure that Unknown/Null is still handled as expected? Seems we dont have any tests for that (because my change passed all the tests)? If we need to create a test, where should I create?

Please tell me more details as this is my first time contributing to the project and I'm not familiar with it yet.

Reply:
Yes, Unkown/Null type has javaType of boolean.class. https://github.com/prestosql/presto/blob/master/presto-main/src/main/java/io/prestosql/type/UnknownType.java#L35
Maybe just add another if/else branch to handle this case.

Reply:
Added type cast for others.

Binary, float, double work for Hive/Spark.

Only double works for Presto due to syntax differences.

Reply:
Reverted the binary check to old code for safety. So no regression.